### ISSUE TITLE
Strip HTML tags from Calm data

### ISSUE THREAD

**Issue by jennpb** [wellcomecollection/platform#4721]
We've got rogue HTML tags displaying on Calm-derived works in places where it's disruptive to legibility.

I propose that we strip HTML tags from these specific fields:

work title:  https://wellcomecollection.org/works/zqqh37sb
subject labels: https://wellcomecollection.org/works/ajrejevs

**Comment by jennpb** [comment]
Just to add that the reason I don't think it's a good idea to render the html in titles is because titles can be quite long, and italics are distruptive to legibility as well. Better just to render as normal text.

**Comment by warrd** [comment]
Worth mentioning currently in Calm we have `description`, `physicalDescription` and `notes` where we allow certain html (and normalise it so always closing tags etc). cc @jtweed

**Comment by jennpb** [comment]
Yes, and those should be rendered as previously agreed. No change there!

**Comment by warrd** [comment]
Cool no probs, wasn't saying we did just pointing out so we have all the info in one place.

**Comment by warrd** [comment]
The works above are rendering now. However the title bar displays the `<i>` tags, so not sure how we should deal with that :thinking:  @gestchild what do you think?
